src/argilla/client/feedback/schemas.py

```diff
@@ -485,3 +485,15 @@ def update_settings(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         if isinstance(values.get("values"), list):
             values["settings"]["options"] = [{"value": value, "text": value} for value in values.get("values")]
         return values
+
+    @property
+    def __all_labels__(self):
+        return [entry["value"] for entry in self.settings["options"]]
+
+    @property
+    def __label2id__(self):
+        return {label: idx for idx, label in enumerate(self.__all_labels__)}
+
+    @property
+    def __id2label__(self):
+        return {idx: label for idx, label in enumerate(self.__all_labels__)}
```

Comment:
Just to have more details on this, why is this useful here if we can have ties/draws, unassigned values to ranks, and some other special cases for the `RankingQuestion`?

Reply:
When having ties draws etc. we currently include the first one.